### ISSUE TITLE
uppy fixes

### ISSUE THREAD

**Issue by WhoAteDaCake** [SpotlightData/nanowire-extensions#27]
* Fix uppy file table miss spelling, sumbitRender -> submitRender
* Rename OneDrive -> OneDriveUppy
* Improve onedrive columns set up (export bytes to readable functions etc)

**Comment by WhoAteDaCake** [comment]
In uppy onedrive getFiles
> Uncaught ReferenceError: receiver is not defined

**Comment by WhoAteDaCake** [comment]
add removeAll pending files button

**Comment by WhoAteDaCake** [comment]
main.bundle.js:23481 Uncaught (in promise) TypeError: Cannot read property 'catch' of undefined
```javascript
      var upload = files.map(function (file) {
        var temp = file.name.split('.');
        var ext = temp[temp.length - 1].toLowerCase();

        if (!_this.opts.allowedExtensions.includes(ext)) {
          return Promise.resolve({
            name: file.name,
            type: file.type,
            e: 'File type not allowed'
          });
        }

        return _this.uppy.addFile({
          source: 'Local',
          name: file.name,
          type: file.type,
          data: file
        }).catch(function (e) {
          return {
            name: file.name,
            type: file.type,
            e: e.message
          };
        });
      }); // If error is not cought, it returns nothing, so it can be filtered out
```

**Comment by WhoAteDaCake** [comment]
Fixed in [#28](https://github.com/SpotlightData/nanowire-extensions/pull/28)
